docs/configuration.txt

```diff
@@ -43,6 +43,138 @@ will overwrite the prior one:
  3. options from config file
  4. command-line properties
 
+Table Settings
+==============
+
+.. _number_of_replicas
+
+**number_of_replicas**
+  | *Default:*    ``1``
+  | *Runtime:*    ``yes``
+
+.. _refresh_interval
+
+**refresh_interval**
+  | *Default:*   ``1000``
+  | *Runtime:*   ``yes``
+
+  Specifies the refresh interval of a shard in milliseconds.
+
+Blocks
+------
+
+.. _read_only
+
+**read_only**
```

Comment:
the setting names must be as they appear in elasticsearch / crate.yml
e.g.
`index.blocks.read_only`

applies to all following settings